lib/graphql/language/lexer.rb

```diff
@@ -1004,7 +1004,7 @@ def self.record_comment(ts, te, meta)
       def self.emit(token_name, ts, te, meta)
         meta[:tokens] << token = GraphQL::Language::Token.new(
           name: token_name,
-          value: meta[:data][ts...te].pack("c*"),
+          value: meta[:data][ts...te].pack("c*").force_encoding("UTF-8"),
```

Comment:
Nevermind, that screws up some unicode BOM parsing. Tried it and got some failing tests.